File: app/src/main/java/eu/toma/dev/playground/warcraft/model/item/Spell.java
package eu.toma.dev.playground.warcraft.model.item;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Spell {

    @SerializedName("id")
    @Expose
    public Integer id;
    @SerializedName("name")
    @Expose
    public String name;
    @SerializedName("icon")
    @Expose
    public String icon;
    @SerializedName("description")
    @Expose
    public String description;
    @SerializedName("castTime")
    @Expose
    public String castTime;

}

File: app/src/main/kotlin/warcraft/model/item/Spell.kt
package warcraft.model.item

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Spell {

    @SerializedName("id")
    @Expose
    var id: Int? = null
    @SerializedName("name")
    @Expose
    var name: String
    @SerializedName("icon")
    @Expose
    var icon: String
    @SerializedName("description")
    @Expose
    var description: String
    @SerializedName("castTime")
    @Expose
    var castTime: String

}